package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderResult.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Christian Mathis.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Christian Mathis - author of some PRs
+ *******************************************************************************/
+package de.dentrassi.pm.jenkins;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Results of the Upload operation
+ * 
+ * @author Christian Mathis
+ */
+public class UploaderResult implements Serializable
+{
+    private static final long serialVersionUID = -3089286880912224513L;
+
+    /*
+     * Map containing the id and filename of the successfully uploaded artifacts

```

Comment:
unless field is public (standatd javadoc generation) move this to the getter/setter or does not appear